sort3.jsx

```diff
@@ -0,0 +1,2 @@
+var number = [19, 3, 81, 1, 24, 21];
```

Comment:
```suggestion
const  number = [19, 3, 81, 1, 24, 21];
```
varの定義方法は一昔前の定義になります。
letかconstで定義するようにしてください。